src/util/ClientServiceUtility.java

```diff
@@ -0,0 +1,60 @@
+package util;
+
+import entities.BaseEntity;
+import entities.User;
+import java.util.Optional;
+
+public final class ClientServiceUtility {
+  public static BaseEntity updateBaseEntity(BaseEntity baseEntityToUpdate, BaseEntity updated) {
```

Comment:
Rename to args to `target` and `source`

Reply:
Rename the function to `fetchBaseEntity`

Reply:
Move this to another file `BaseEntityUtil`

Reply:
solved